openrtb2/video.go

```diff
@@ -112,8 +187,9 @@ type Video struct {
 	//   where 0 = no, 1 = yes.
 	//   If a bidder sends markup/creative that is itself skippable, the
 	//   Bid object should include the attr array with an element of
-	//   16 indicating skippable video. Refer to List 5.3.
-	Skip *int8 `json:"skip,omitempty"`
+	//   16 indicating skippable video. Refer to List: Creative
+	//   Attributes in AdCOM 1.0.
+	Skip int8 `json:"skip,omitempty"`
```

Comment:
Ah, thanks for fixing this. The list reference threw me off. Should it still be a pointer?

Reply:
Indeed, it was pointer, will bring it back...